<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimingDialog</class>
 <widget class="QDialog" name="TimingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="spin_fps">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spin_seconds">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string>&quot;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Frames</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spin_frames">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Layers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="check_layer_keep">
        <property name="text">
         <string>Keep Initial Timing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="check_layer_trim">
        <property name="text">
         <string>Trim Extra Frames</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="check_layer_scale">
        <property name="text">
         <string>Scale Timing</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>TimingDialog</receiver>
   <slot>btn_clicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_fps</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TimingDialog</receiver>
   <slot>changed_fps(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_seconds</sender>
   <signal>valueChanged(double)</signal>
   <receiver>TimingDialog</receiver>
   <slot>changed_seconds(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_frames</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TimingDialog</receiver>
   <slot>changed_frames(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_clicked(QAbstractButton*)</slot>
  <slot>changed_fps(double)</slot>
  <slot>changed_seconds(double)</slot>
  <slot>changed_frames(int)</slot>
 </slots>
</ui>
